README.md

```diff
@@ -4,8 +4,6 @@ Terraform 0.13. Pin module version to ~> 3.X. Submit pull-requests to master bra
 
 Terraform 0.12. Pin module version to ~> 2.X. Submit pull-requests to terraform012 branch.
```

Comment:
Might want to update or add and except for 0.13 and above.

Reply:
thansk for the catch! I added to reflect this!